src/main/java/com/autotune/analyzer/recommendations/engine/RecommendationEngine.java

```diff
@@ -123,17 +126,44 @@ private static int getNumPodsForNamespace(Map<Timestamp, IntervalResults> filter
         return (int) Math.ceil(max_pods_cpu);
     }
 
-    private void init() {
-        // Add new models
+    private void DefaultLoadRecommendationModel() {
+        // create both cost and performance model by default
         recommendationModels = new ArrayList<>();
         // Create Cost based model
         CostBasedRecommendationModel costBasedRecommendationModel = new CostBasedRecommendationModel();
-        // TODO: Create profile based model
         registerModel(costBasedRecommendationModel);
         // Create Performance based model
         PerformanceBasedRecommendationModel performanceBasedRecommendationModel = new PerformanceBasedRecommendationModel();
         registerModel(performanceBasedRecommendationModel);
-        // TODO: Add profile based once recommendation algos are available
+    }
+
+    private void DefaultLoadRecommendationModelForAutoAndRecreate() {
+        // create performance model by default
+        recommendationModels = new ArrayList<>();
+        // Create Performance based model
+        PerformanceBasedRecommendationModel performanceBasedRecommendationModel = new PerformanceBasedRecommendationModel();
+        registerModel(performanceBasedRecommendationModel);
+    }
+
+    private void CustomLoadRecommendationModel(List<String> modelName) throws InvalidModelException {
+        // Add new models
+        recommendationModels = new ArrayList<>();
+        for(String model : modelName){
```

Comment:
Add spaces here after `for` and at the end before the braces start.
Kindly re-visit all the statements and cross check for the same.